docs/pages/distribution/publishing-websites.mdx

```diff
@@ -3,12 +3,56 @@ title: Publish websites
 description: Different ways to publish the Expo web app with third-party services.
 ---
 
-import { Terminal } from '~/ui/components/Snippet';
+import { APIBox } from '~/components/plugins/APIBox';
 import { Step } from '~/ui/components/Step';
 import { Tab, Tabs, TabsGroup } from '~/ui/components/Tabs';
+import { Table, TableHead, Row, HeaderCell } from '~/ui/components/Table';
+import { Terminal } from '~/ui/components/Snippet';
+import { YesIcon, NoIcon, PendingIcon } from '~/ui/components/DocIcons';
 
 A web app created using Expo can be served locally for testing out the production behavior. Once the testing phase checks out, you can choose from a variety of third-party services to host it.
 
+## Output targets
+
+Expo Router supports three output targets for web apps. The `web.output` target can be configured in the **app.json** file:
+
+```json app.json
+{
+  "expo": {
+    "web": {
+      "output": "server",
+      "bundler": "metro"
+    }
+  }
+}
+```
+
+> If you are using legacy Expo Webpack (deprecated), only `single` is supported.
+
+| Output             | Expo Router | API Routes  |  Description                                                                                                                   |
+| ------------------ | ----------- | ----------- |  ----------------------------------------------------------------------------------------------------------------------------- |
+| `single` (default) | <YesIcon /> | <NoIcon />  |  Outputs a Single Page Application (SPA), with a single index.html in the output folder, and has no statically indexable HTML. |
+| `server`           | <YesIcon /> | <YesIcon /> |  Creates a client & server folder. Client files are outputed as seperate HTML files. API routes as seperate JavaScript files   |
```

Comment:
```suggestion
| `server`           | <YesIcon /> | <YesIcon /> |  Creates **client** and **server** directories. Client files are output as separate HTML files. API routes as separate JavaScript files for hosting with a custom Node.js server.   |
```

Added a bit more context for the API routes part of this description based on https://docs.expo.dev/versions/latest/config/app/#output. It wasn't clear what would one do with API routes here or why they should know about it.